.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.lock
 vendor
 build
+.idea
```

Comment:
this has nothing to do with the pull request you proposed, and on top of that it isn't nice to pollute someone's `.gitignore` with files that are irrelevant to them. (abandon IDEs, come to Vim 🤣 )